study/src/test/java/reflection/ReflectionTest.java

```diff
@@ -65,11 +72,11 @@ void givenClass_whenGetsAllConstructors_thenCorrect() {
     void givenClass_whenInstantiatesObjectsAtRuntime_thenCorrect() throws Exception {
         final Class<?> questionClass = Question.class;
 
-        final Constructor<?> firstConstructor = null;
-        final Constructor<?> secondConstructor = null;
+        final Constructor<?> firstConstructor = questionClass.getDeclaredConstructors()[0];
+        final Constructor<?> secondConstructor = questionClass.getDeclaredConstructors()[1];
```

Comment:
[0] [1] 보다는 타입을 지정하는게 더 좋을 것 같아요~
```suggestion
        final Constructor<?> firstConstructor = questionClass.getDeclaredConstructor(String.class, String.class, String.class);
```

Reply:
오 이런 방법이 있군요!
확실히 더 직관적인 코드가 되네요! 반영하겠습니다